ui/src/reportWebVitals.js

```diff
@@ -1,6 +1,8 @@
-const reportWebVitals = onPerfEntry => {
```

Comment:
What does this do? I am not familiar with web vitals?

Reply:
it came with `npx create-react-app`, I think it's something they added recently

Reply:
https://create-react-app.dev/docs/measuring-performance/
I will remove it since we don't plan to collect telemetry